selfdrive/car/subaru/values.py

```diff
@@ -119,6 +119,10 @@ class CAR(Platforms):
       SubaruCarInfo("Subaru XV 2018-19", video_link="https://youtu.be/Agww7oE1k-s?t=26"),
     ],
     specs=CarSpecs(mass=1568, wheelbase=2.67, steerRatio=15),
+    test_routes=[
+      "791340bc01ed993d|2019-03-10--16-28-08",
```

Comment:
I think most of the test routes should be automatically generated, and then we just inject a couple of extra cases here when we have special cases to test